pom.xml

```diff
@@ -47,6 +45,20 @@
               <transformationSet>
                 <dir>.</dir>
                 <includes>
+                  <include>indexing.rdf</include>
```

Comment:
This section depends on an `rdfs2html.xsl` file, which was not included in the commit. Should I revert that change or simply include the XSLT from @escowles ?